src/Services/ReorderElements.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+namespace DNADesign\Elemental\Services;
+
+use InvalidArgumentException;
+use SilverStripe\Core\Convert;
+use SilverStripe\ORM\DataObject;
+use SilverStripe\ORM\Queries\SQLUpdate;
+use DNADesign\Elemental\Models\BaseElement;
+
+class ReorderElements
+{
+    /**
+     * @var BaseElement $block
+     */
+    private $element;
```

Comment:
We always prefer protected over private

Reply:
+1 this. Private makes it unnecessarily difficult to extend.